src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -225,6 +214,57 @@ citus_relation_size(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * GetShardStatisticsInParallel generates query lists for obtaining shard
+ * statistics and then sends the commands in parallel by opening connections
+ * to available nodes. It returns the connection list.
+ */
+List *
+GetShardStatisticsInParallel(List *citusTableIds, bool includeMinMax)
```

Comment:
Usually a boolean is used as a subtle thing, like changing a parameter while generating the query maybe. Here this parameter is doing more than that, do you see a way of removing this parameter and having a dedicated method for the case where we want shardMinMax values?

I would prefer simplicity over performance where performance isn't that important